corehq/apps/app_manager/tests/util.py

```diff
@@ -113,6 +113,10 @@ def parse_normalize(xml, to_string=True, is_html=False):
 
 def _check_shared(expected, actual, checker, extension):
     # snippet from http://stackoverflow.com/questions/321795/comparing-xml-in-a-unit-test-in-python/7060342#7060342
+    if isinstance(expected, six.binary_type):
```

Comment:
nit: was this an automatic change by a modernizer script? If not `bytes` means the same thing on both Python 2 and 3, so is preferable to `six.binary_type`.

Reply:
This was a manual change.  Good point about ```six.binary_type``` - I'll be conscious of that going forward, and do a cleanup at some point.

Reply:
https://trello.com/c/RUI4xVF3/68-sixbinarytype-bytes